src/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,26 @@
+import classNames from 'classnames';
+
+type PropsType = {
+  htmlFor: string;
+  children: React.ReactNode;
```

Comment:
Используется React, а его импорта нет. 
```suggestion
  children: ReactNode;
```